.github/workflows/rebuild-100k.yml

```
@@ -0,0 +1,57 @@
+name: Rebuild 100k sample
+
+on:
+  # cron job once a week on mondays
+  schedule:
+    - cron:  '* * * * 1'
```

This schedule is actually every minute on Mondays. :grimacing: We want something like this instead:

```suggestion
  # cron job once a week on Mondays at 12:42 UTC
  schedule:
    - cron:  '42 12 * * 1'
```

You might even be able to use `Mon` instead of `1`, but GitHub's docs aren't clear if they support that or not (though there's a suggestion they do).

---

Ha! Good catch. Thanks